azure-ts-dynamicresource/cdnCustomDomain.ts

```diff
@@ -0,0 +1,244 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+
+import * as msRestAzure from "@azure/ms-rest-nodeauth";
+import * as cdnManagement from "@azure/arm-cdn";
+import { ServiceClientCredentials } from "@azure/ms-rest-js";
+
+export interface CustomDomainOptions {
+  resourceGroupName: string;
```

Comment:
nit: you have 2-space indent on.  But i think all our sdk/example code is 4-space.